src/components/transactions/Stake/StakeModalContent.tsx

```diff
@@ -61,8 +61,8 @@ export const StakeModalContent = ({ stakeAssetName, icon }: StakeProps) => {
   // staking token usd value
   const amountInUsd =
     Number(amount) *
-    (Number(normalize(stakeData?.stakeTokenPriceEth || 1, 18)) /
-      Number(normalize(stakeGeneralResult?.usdPriceEth || 1, 18)));
+    (Number(normalize(stakeData?.stakeTokenPriceEth || 1, 18)) *
```

Comment:
I think this should stay as divide by

Reply:
![ClaimUSD](https://github.com/aave/interface/assets/26660211/7895ded2-3a64-4696-8387-a006cf41580a)
![UnstakeUSD](https://github.com/aave/interface/assets/26660211/7c542507-82f4-42e0-83a1-36edad7f6e28)

Fixed in both places